code/addons/docs/src/preset.ts

```diff
@@ -27,47 +23,24 @@ const getResolvedReact = async (options: Options) => {
     // addon-docs, causing addon-docs's dependencies not to be hoisted.
     // This might also affect regular users who have a similar setup.
     // Explicitly alias @mdx-js/react to avoid this issue.
-    mdx: resolvedReact.mdx ?? dirname(require.resolve('@mdx-js/react/package.json')),
+    mdx: resolvedReact.mdx ?? dirname(require.resolve('@mdx-js/react')),
```

Comment:
why did you remove `/package.json` here?

Reply:
Because @mdx-js/react v3 has an exports map which does not contain package.json as an entry